vic/drivers/shared_all/src/generate_default_state.c

```diff
@@ -53,15 +53,23 @@ generate_default_state(all_vars_struct *all_vars,
     };
     double                   Cv;
     double                   tmp;
+    double                   AreaFactor;
+    double                   TreeAdjustFactor = 1.;
+    double                   lakefactor = 1.;
+    double                   albedo_sum;
     double                ***tmpT;
     double                 **tmpZ;
     int                      ErrorFlag;
 
     cell_data_struct       **cell;
     energy_bal_struct      **energy;
+    veg_var_struct         **veg_var;
+    gridcell_avg_struct      gridcell_avg;
```

Comment:
This should be a pointer. You don't want to copy the whole structure, you just want to point to the structure